getting_started/workflow/project_setup/project_organization.rst

```diff
@@ -51,12 +51,12 @@ to store built levels that use them.
 Importing
 ---------
 
-Godot version previous to 3.0 did the import process from files outside
+Godot versions prior to 3.0 did the import process from files outside
 the project. While this can be useful in large projects, it
 resulted in an organization hassle for most developers.
 
-Because of this, assets are now imported from within the project
-folder transparently.
+Because of this, assets are now transparently imported from within the project
+folder.
 
-If a folder shouldn't be imported into Godot an exception can be made with a
-.gdignore file.
+If a folder shouldn't be imported into Godot, an exception can be made with a
+.gitignore file.
```

Comment:
Actually, `.gdignore` is the correct filename. This sentence reference's *Godot's import process*, not something with Git.